cachi2/core/config.py

```diff
@@ -27,3 +27,14 @@ def get_worker_config():
         config = Config()
 
     return config
+
+
+def set_config(path: str) -> None:
+    """Set global config variable using input from file."""
+    global config
+
+    try:
+        with open(path, "r") as file:
+            config = Config(**yaml.safe_load(file))
```

Comment:
This will still fail with an untreated error in case the yaml file carries a single string (it's valid yaml :confounded:)
```
TypeError: cachi2.core.config.Config() argument after ** must be a mapping, not str
```

Maybe we can do the yaml handling in `cli.py` and pass just a dict to the `set_config` function?

Reply:
Or use `Config.parse_obj`, which will fail with a nicer error message if the "obj" is not a dict